CHANGELOG.md

```diff
@@ -15,3 +15,56 @@ and this project adheres to
 ## [v0.0.2] - 2025-04-23
 
 - move to a callback for SNMP get requests
+
+## [v0.0.1] - 2025-04-27
+
+- Tested the MIB's and library by running an SNMP walk. This command iterates through all entries, starting at a certain point, eg. '1':
+  Command: `snmpwalk -v2c -c public 192.168.2.17 1`
+  It provides an interesting extra test.
+
+## [v0.0.1] - 2025-04-24
+
+- #10 from remove_compiler_warning_changes
+- Remove the compiler warning from CMakeList.txt
+
+## [v0.0.1] - 2025-04-22
```

Comment:
Perhaps CL entries prior to v0.0.1 could just have the date, not the version. If you agree, perhaps make that change on a future PR.